### DISCUSSION TITLE
ETA next release

### DISCUSSION THREAD

**Discussion by pedropgusmao** [omry/omegaconf#868] [General]
Hi, I see that OmegaConf has included quite a few updates since its current release, including support for `Bytes` =). 
I was wondering if there is an ETA for version 2.2 coming up? 
Thanks

**Comment by Jasha10** [comment]
Hi @pedropgusmao!
We are planning to publish OmegaConf 2.2 at the same time as Hydra 1.2. Our self-imposed deadline is mid-April.
I will publish an OmegaConf 2.2 dev release to pypi shortly so that it will be easier to test out the newest OmegaConf features.

**Reply by pedropgusmao** [reply]
Nice! Thanks for this.

**Reply by Jasha10** [reply]
FYI the `2.2.0.dev1` development release is now available on pypi, including support for `bytes`.
```text
pip install omegaconf==2.2.0.dev1
```

**Reply by jieru-hu** [reply]
thank you @Jasha10  😃

**Reply by pedropgusmao** [reply]
Thanks, @Jasha10 !
Out of curiosity, does Hydra also have a `.dev` version that would work with `omegaconf==2.2.0.dev1` ?

**Reply by Jasha10** [reply]
Yes, you can install Hydra versions `1.2.0.dev1` and `1.1.2.dev0` from pypi; see [the pypi release history for `hydra-core`](https://pypi.org/project/hydra-core/#history).
